soccer/planning/Path.hpp

```diff
@@ -227,6 +239,14 @@ class AngleFunctionPath : public Path {
 
     virtual RJ::Time startTime() const override { return path->startTime(); }
     virtual void setStartTime(RJ::Time t) override { path->setStartTime(t); }
+    virtual void setDebugText(QString string) override {
+        path->setDebugText(std::move(string));
+    }
+
+    virtual void drawDebugText(SystemState* state, const QColor& color = Qt::darkCyan,
```

Comment:
Ahh gotcha, I missed that.  Should they also set the debug text for the angle path itself or just the inner x/y-path?